ptf/sairoute.py

```diff
@@ -1050,10 +1089,20 @@ def gleanEndForwardTest(self):
             sai_thrift_remove_neighbor_entry(self.client, nbr_entry2)
```

Comment:
local variable nbr_entry1 referenced before assignment

Reply:
fixed in https://github.com/ravi861/SAI/pull/3 and https://github.com/ravi861/SAI/pull/2